### ISSUE TITLE
Sobre persona objeto de la ley

### ISSUE THREAD

**Issue by genesis3g** [cslve/reglamento-conati#10]
En el numeral 10 del artículo 2, dice:
"Las sociedades de cualquier naturaleza, las fundaciones, empresas, asociaciones civiles y las demás creadas con fondos públicos o dirigidas por las personas a las que se refiere este artículo, en las que ellas designen sus autoridades, o cuando los aportes presupuestarios o contribuciones en un ejercicio efectuados por las personas referidas en el presente artículo representen el cincuenta o más de su presupuesto".

Considero que en este punto se debe ser muy específicos sobre este tipo de personas a las que se refiere la ley, y más aún si este tipo de persona jurídica ofrece bienes y servicios para la APN. 

Del mismo modo, los productos de investigaciones del sector universitario público, también deben regirse bajo esta normativa.

De ello me surge la interrogante, ¿el sector privado que ofrezca bienes y servicios para la AP, debería también regirse por este reglamento?

**Comment by ghostbar** [comment]
A tu pregunta: no, pues los contratos no pueden ser considerados "aportes presupuestarios o contribuciones".

Sí, esto hace que las universidades entren en esto mientras el aporte presupuestario sea mayor al 50%.

**Comment by genesis3g** [comment]
Y así no sea mayor, entra. Son objeto de ley directo. 

Más bien me refería a esto: al producto de una investigación y a la investigación misma. El investigador pudiese decir que la ley es para los administrativos, los docentes, etc, no para quien investiga.

**Comment by ghostbar** [comment]
Claro, yo me refería a las universidades en general, no sólo las públicas; hay algunas privadas que reciben aportes presupuestarios grandes del Estado.

Sobre las investigaciones: supongo que sí, con tal y sea software.

**Comment by genesis3g** [comment]
De igual manera, reformulo mi pensamiento:
Si una empresa del sector privado va a trabajar para la AP en un momento determinado, para algo puntual, de esa relación que surja, el SP debe entregar lo que vaya a entregar, en un formato al menos, interoperable. ¿Me explico?

**Comment by genesis3g** [comment]
Recuerda que no solo es software. Allí entra lo que sería el uso de estándares abiertos para documentos, imágenes, etc.

**Comment by ghostbar** [comment]
Eso debe exigírsele en el contrato como consecuencia de lo que se usa en la AP.

Sí, ahora sí entiendo a lo que te refieres.

**Comment by genesis3g** [comment]
Yo creo que debe quedar claro en este reglamento. If you know what I mean.
Hay un conjunto de leyes (debo revisar la de contrataciones públicas a ver si dice algo) que pudiesen chocar con LIG en ese sentido o uno similar.

**Comment by jorgescalona** [comment]
El sector privado debe usar formatos abiertos para intercambiar la info con la apn, y el ámbito de la ley también incluye a las fundaciones (subsidiadas por el gobierno nacional), colegios católicos x ejemplillo, fe y alegria, etc etc
